File: Java/Warmup-2/stringBits.java
/*
Given a string, return a new string made of every other char starting with the first, so "Hello" yields "Hlo".


stringBits("Hello") → "Hlo"
stringBits("Hi") → "H"
stringBits("Heeololeo") → "Hello"
*/

class stringBits{
    public String stringBits(String str) {
        String newString = "";
        for (int i = 0; i<str.length(); i+=2){
          newString = newString + str.charAt(i);
        }
        return newString;
      }      
}

File: CodingBat/Python/Warmup-2/string_bits.py
"""
Given a string, return a new string made of every other char starting with the first, so "Hello" yields "Hlo".

string_bits('Hello') → 'Hlo'
string_bits('Hi') → 'H'
string_bits('Heeololeo') → 'Hello'

@author unobatbayar
"""
def string_bits(str):
  new = ''
  for i in range(len(str)):
    if i % 2 == 0:
      new = new + str[i]
  return new

